srcpkgs/sof-bin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'sof-bin'
+pkgname=sof-bin
+version=1.5.1
+revision=1
+archs="i686 x86_64"
+wrksrc=${pkgname}-stable-v${version}
+make_dirs="/usr/lib/firmware/intel/sof/intel-signed 0755 root root
+ /usr/lib/firmware/intel/sof/public-signed 0755 root root
+ /usr/lib/firmware/intel/sof-tplg 0755 root root"
+short_desc="Sound Open Firmware and topology binaries"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="custom:Intel, custom:NXP"
+homepage="https://thesofproject.github.io/latest/index.html"
+distfiles="https://github.com/thesofproject/sof-bin/archive/stable-v${version}.tar.gz"
+checksum=270125fb621b9919de4190f53a86de5dc1163dcf839c349f4b22724a5a9c1fb1
+
+do_install() {
+	local intel_path="lib/firmware/intel"
+	for f in ${intel_path}/sof/v${version}/*.{ldc,ri}; do
+		vinstall ${f} 0644 /usr/${intel_path}/sof
+	done
+	for f in ${intel_path}/sof/v${version}/intel-signed/*; do
+		vinstall ${f} 0644 /usr/${intel_path}/sof/intel-signed
+	done
+	for f in ${intel_path}/sof/v${version}/public-signed/*; do
+		vinstall ${f} 0644 /usr/${intel_path}/sof/public-signed
+	done
+	ln -s ${DESTDIR}/usr/${intel_path}/sof/sof-bdw-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-bdw.ri
```

Comment:
`ln -s sof-bdw-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-bdw.ri`

Do this for all the commands below, where applicable, and, if possible, use a for loop.

Reply:
I've updated the template to use this format with a for loop. There are a couple where the architecture in the middle (bdw in your example) doesn't match the symlink, so I had to leave them out of the for loop.